prime-router/metadata/fhir_mapping/hl7/resource/DiagnosticReport.yml

```diff
@@ -200,4 +210,74 @@ extension_2:
         system: SYSTEM_URL, $system_code
         code: String, OBR.25
       constants:
-        system_code: v2-0123
\ No newline at end of file
+        system_code: v2-0123
+
+extension_3:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: parent-observation-sub-id
+    valueString:
+      type: STRING
+      valueOf: OBR.26.2
+      expressionType: HL7Spec
+
+extension_4:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: parent-observation-value-descriptor
+    valueString:
+      type: STRING
+      valueOf: OBR.26.3
+      expressionType: HL7Spec
+
+extension_5:
```

Comment:
I put the OBR 29 values into extensions so that they could be distinguished from the identifiers above.

Reply:
OBR.29.1 doesn't exist in the HCA example but I added it just in case.